app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -127,10 +131,6 @@ class GleanMetricsService(private val context: Context) : MetricsService {
     }
 
     override fun shouldTrack(event: Event): Boolean {
-        return Settings.getInstance(context).isTelemetryEnabled && event.wrapper != null
```

Comment:
Do we still want to track events if Telemetry is disabled, even if we're not uploading?

Reply:
@colintheshots We do this check in Metrics controller before this code gets called, so it was redundant